packages/fuselage/src/components/Message/MessageMetrics/__snapshots__/MessageMetrics.spec.tsx.snap

```diff
@@ -117,7 +117,7 @@ exports[`[MessageMetrics Rendering] renders Default without crashing 1`] = `
                 class="rcx-message-metrics__content-wrapper"
               >
                 <div
-                  class="rcx-message-metrics__item"
+                  class="rcx-message-metrics__item undefined"
```

Comment:
let's filter the truth's classNames to avoid this undefined value on the classes